src/tz.cpp

```diff
@@ -92,9 +94,6 @@
 #  define TARGET_OS_SIMULATOR 0
 #endif
 
-#if USE_OS_TZDB
```

Comment:
Does it hurt to just include this?

Reply:
It caused a failure to build on windows if included. Replaced usages with filesystem. I"ll replace this deletion with a commented out include with info to minimize diff change, unless you have a style preference?

HH is using the fact that windows doesnt have a OS TZDB available to make this work.

Reply:
I would recommend something like

```
if USE_OS_TZDB && !RTC_WINDOWS_FAMILY
```

This serves two purposes. It limits the change to a single line that (hopefullly) won't be changed by upstream and also adds our preprocessor defines, altering a developer reading the code that this is a change we've made. It also makes hunting down our modifications easier in searches of `RTC_`

Reply:
Gotcha, in similar cases where I need to modify these macro blocks to always or never trigger do we have something like RTC_FALSE and RTC_TRUE?

Reply:
These are the list of defines we define globally for all projects https://devtopia.esri.com/runtime/runtimecore/blob/main/docs/writing_code/patterns/preprocessor_defines.md. To get what you'd need, you could just check `if defined(RTC)` as a true for our code, false for other code. If you just need this for specific platforms, then you'd use the platform defines and if just for specific chipsets, you'd use the chipset defines. I hope with all those different combinations, you'd be able to tailor the build very specifically for every use case.

Reply:
Thats awesome, thanks!